layouts/filter.tsx

```diff
@@ -8,15 +8,15 @@ import FilterHeader from "components/molecules/FilterHeader/filter-header";
 
 import useNav from "lib/hooks/useNav";
 
-const FilterLayout = ({children}: {children: React.ReactNode}) => {
-
+const FilterLayout = ({ children }: { children: React.ReactNode }) => {
+  
   const { toolList, selectedTool, filterName, selectedFilter, userOrg } = useNav();
 
   return (
     <>
       <TopNav />
       <div className="page-container flex min-h-[calc(100vh-(54px+95px))] flex-col items-center">
-        <div className="info-container min-w-full min-h-[100px]">
+        <div className="info-container w-full min-h-[100px]">
```

Comment:
the reason for this change is that a parent element with no fixed width will not allow the children elements with overflow auto or scroll properties to flow properly, causing the parent element to go over the screen.